Cargo.toml

```
@@ -10,25 +10,44 @@ categories = ["web-programming::http-client", "network-programming"]
 keywords = ["http", "client", "request"]
 edition = "2018"
 
+#[dependencies.native-tls]
+#version = "^0.2"
+#optional = true
+
 [dependencies]
-unicase = "^2.6"
+# sgx dependencies
+#sgx_tstd = { rev = "v1.1.3", git = "https://github.com/apache/teaclave-sgx-sdk.git", optional = true ,features = ["net", "untrusted_time"]}
+sgx_tstd = { version = "1.1.4", git = "https://github.com/haerdib/incubator-teaclave-sgx-sdk", branch = "master",optional = true ,features = ["net", "untrusted_time"]}
```

The spacing is off here:
```suggestion
sgx_tstd = { version = "1.1.4", git = "https://github.com/haerdib/incubator-teaclave-sgx-sdk", branch = "master", optional = true, features = ["net", "untrusted_time"] }
```